net/dns/resolver/tsdns.go

```diff
@@ -196,6 +196,7 @@ type Resolver struct {
 	mu           sync.Mutex
 	localDomains []dnsname.FQDN
 	hostToIP     map[dnsname.FQDN][]netip.Addr
+	wcHostList   []dnsname.FQDN
```

Comment:
`wc` is a little foreign to me here, I'd find something like `wildcards` or `wildDomains` faster to interpret